livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -161,6 +164,95 @@ def with_cerebras(
             temperature=temperature,
         )
 
+    @staticmethod
+    def with_vertexai(
```

Comment:
nit: We took a decision a while ago about not adding ai at the end of every brand (only for those who aren't obvious)
```suggestion
    def with_vertex(
```